packages/config-array/tests/config-array.test.js

```diff
@@ -1776,10 +2364,10 @@ describe("ConfigArray", () => {
 						"node_modules/foo/bar.js",
 					);
 
-					assert.strictEqual(configs.isFileIgnored(filename), true);
+					assert.strictEqual(configs.isFileIgnored(filename), false);
```

Comment:
Yeah, this test seems wrong. That file should not be ignored by those ignore patterns. Also, patterns should not start with `/`.

By the description of the test ("should return true when a subdirectory is ignored and then we try to unignore a directory"), I'd guess that the patterns should have been `ignores: ["**/node_modules/*", "!node_modules/"]`.

Reply:
I see, that makes sense. It looks like the problem with all these tests is that they're using an ignore pattern like `"/**/node_modules/..."`, but since the `node_modules` directory is at the top level, that pattern no longer matches.

Reply:
Fixed in 0525daa3fce77db289ba6af2745a3d9dc7891adb, thanks!